appcoins-ads/src/main/java/com/asf/appcoins/sdk/ads/poa/manager/PoAManager.java

```diff
@@ -374,12 +376,16 @@ private void startCampaign(Campaign campaign) {
     initiateProofSending();
   }
 
-  private void promptWalletInstall() {
-    Log.d(TAG,"Prompting Wallet Install");
-    if(!WalletUtils.isDialogVisible()){
+  private void promptWalletNotification() {
+    if (!showPopUpNotification) {
+      Log.d(TAG, "Prompting Notification Install");
+      preferences.edit()
```

Comment:
Maybe consider having a method that gets and updates the value, to avoid having use cases when the local variable is updated and the preferences are not or the other way around.